src/history/history.ts

```diff
@@ -117,7 +194,7 @@ export class HistoryList {
   }
 
   public show_error(error: string): void {
-    // TODO
+    // to do
```

Comment:
delete whole method?

Reply:
I'd like to leave this in place.... and actually use it - see #12